getCurrentSushiswapLiquidity.ts

```diff
@@ -0,0 +1,31 @@
+const { ethers } = require('ethers');
+const jsonABI = require("./abi/gohm-eth-abi.json");
+
+const getCurrentSushiswapLiquidity = async(pairAddress, tokenAddress) => {
+    let tokenLiquidity = "";
+    const provider = new ethers.providers.JsonRpcProvider("https://arb1.arbitrum.io/rpc");
+    const contract = new ethers.Contract(
+        pairAddress,
+        jsonABI.result,
+        provider
+    );
+    try{
+        const token0 = await contract.token0();
+        const token1 = await contract.token1();
+        const result = await contract.getReserves();
+        if(tokenAddress === token0){
+            tokenLiquidity = ethers.utils.formatUnits(result._reserve0, 18);
```

Comment:
@math-marcellino untuk decimalsnya ini gabisa di hardcode mathew. nanti tergantung tokennya.

bisa ambil decimalsnya melalui ini `ERC20(tokenAddress).decimals()`

Reply:
Oke mas, let me fix this thank youu